### ISSUE TITLE
External URL config missing

### ISSUE THREAD

**Issue by tjorim** [home-assistant/frontend#12363]
### Checklist

- [X] I have updated to the latest available Home Assistant version.
- [X] I have cleared the cache of my browser.
- [X] I have tried a different browser to see if it is related to my browser.

### Describe the issue you are experiencing

I try to set up the external URL of my Home Assistant instance. However the option is not showing up.

### Describe the behavior you expected

I can set the Home Assistant URL under the general configuration.

### Steps to reproduce the issue

1. Enable advanced mode
2. Go to configuration > settings > general
3. Try to set the external url
...

### What version of Home Assistant Core has the issue?

core-2022.4.5

### State of relevant entities

```txt
Cloud is currently not logged in.
```

### Additional information

I think it happened after logging in and out of Home Assistant Cloud (no active subscription).

Related to https://github.com/home-assistant/frontend/pull/12176?

![Screenshot_20220420-192928](https://user-images.githubusercontent.com/1294876/164291256-c4b8676b-2c0b-45e0-85f4-572df7a9b77d.jpg)

**Comment by tcmoreno** [comment]
Same issue affecting Google Home integration

**Comment by asharma2040** [comment]
Same issue

**Comment by tjorim** [comment]
> Same issue. Have anyone found a solution please.

For now I worked around it by manually configuring it in configuration.yaml:

```
homeassistant:
  external_url: "https://home.assistant.io"
  internal_url: "http://192.168.1.2:8123"
```

**Comment by frenck** [comment]
This issue has been fixed in 2022.5.0b4 (current latest beta).
